packages/apps/human-app/frontend/src/api/services/worker/my-jobs-data.ts

```diff
@@ -14,7 +14,19 @@ const myJobSchema = z.object({
   escrow_address: z.string(),
   chain_id: z.number(),
   job_type: z.string(),
-  status: z.string(),
+  status: z.string().transform((value) => {
+    try {
+      return z
```

Comment:
We can use `z.enum(['ACTIVE', 'CANCELED', 'COMPLETED'])` here

Reply:
Added in new commit